### PR TITLE
ENH: Session id is prefixed with namespace in the k8s environment

### LINKED ISSUE #211: ENH: Session id is prefixed with K8s namespace, when in the K8s environment
Note that the issue tracker is NOT the place for general support.  For
discussions about development, questions about usage, or any general questions,
contact us on https://discuss.xorbits.io/.

### PR DESCRIPTION
**Pull request by ChengjieLi28**
Fixes #211 

## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass

### CODE REVIEW

File: python/xorbits/_mars/services/session/supervisor/core.py [hunk 1]
```diff
@@ -45,9 +45,25 @@ async def __pre_destroy__(self):
             *[mo.destroy_actor(ref) for ref in self._session_refs.values()]
         )
 
-    async def create_session(self, session_id: str, create_services: bool = True):
+    async def _get_namespace(self) -> Optional[str]:
+        envs = await self._cluster_api.get_nodes_info(env=True)
+        for node in envs:
+            env = envs[node].get("env", {})
+            if "k8s_namespace" in env:
+                return env["k8s_namespace"]
```
Comment (aresnow1) [inline python/xorbits/_mars/services/session/supervisor/core.py:53]: Specify `role=NodeRole.SUPERVISOR` to get supervisor's info which is more efficient.

File: python/xorbits/_mars/services/session/supervisor/core.py [hunk 2]
```diff
@@ -45,9 +45,27 @@ async def __pre_destroy__(self):
...
 
-    async def create_session(self, session_id: str, create_services: bool = True):
+    async def _get_namespace(self) -> Optional[str]:
+        envs = await self._cluster_api.get_nodes_info(
+            role=NodeRole.SUPERVISOR, env=True
+        )
+        for node in envs:
+            env = envs[node].get("env", {})
+            if "k8s_namespace" in env:
+                return env["k8s_namespace"]
```
Comment (aresnow1) [inline python/xorbits/_mars/services/session/supervisor/core.py:55]:
```suggestion
        for node_info in envs.values():
            env = node_info.get("env", {})
            if "k8s_namespace" in env:
                return env["k8s_namespace"]
```

**Review by aresnow1 — APPROVED** [review]
LGTM

**Review by UranusSeven — APPROVED** [review]
LGTM

**Pull request merged**